sidecar/sidecar.go

```diff
@@ -93,29 +93,6 @@ func Run(conf config.Config) error {
 
 	appSupervisor := supervisor.NewAppSupervisor(&conf)
 
-	if conf.Log {
-		//Replace the LOGSTASH_REPLACEME string in filebeat.yml with
-		//the value provided by the user
-		//TODO: Make this configurable
-		filebeatConf := "/etc/filebeat/filebeat.yml"
```

Comment:
Also can delete the `/docker/filebeat.yml` file from our repo, and remove its references from the Dockerfiles.